code/controllers/subsystem/garbage.dm

```diff
@@ -200,6 +203,10 @@ SUBSYSTEM_DEF(garbage)
 					to_chat(admin, "## TESTING: GC: -- [ADMIN_VV(D)] | [type] was unable to be GC'd --")
 				#endif
 				I.failures++
+				#ifndef REFERENCE_TRACKING
+				if (ignore_gc_failure)
+					continue
+				#endif
```

Comment:
why the ifndef?

Reply:
the continue would keep code flow from hitting the if check below, and i didn't want to deal with these things conflicting, so i just made it so the compile options overrides it.

Reply:
aight